roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -34,6 +9,11 @@
   when:
   - openshift.common.is_containerized | bool
```

Comment:
What about to set both in the `defaults/main.yml` as:
```yaml
containerized_svc_dir: "{{ '/etc/systemd/system' if openshift.common.is_containerized | bool else '/usr/lib/systemd/system' }}"
ha_svc_template_path: "{{ 'docker-cluster' if openshift.common.is_containerized | bool else 'native-cluster' }}"
```
? This way both variables can be overridden from the inventory file and there is only once place in the role where both variables are set.